pkg/network/config/config_test.go

```diff
@@ -627,33 +615,479 @@ func TestMaxTrackedHTTPConnections(t *testing.T) {
 	})
 
 	t.Run("Not enabled", func(t *testing.T) {
-		newConfig(t)
+		ResetConfig(t)
 		cfg := New()
 		// Default value.
 		require.Equal(t, cfg.MaxTrackedHTTPConnections, int64(1024))
 	})
 }
 
+func TestHTTPMapCleanerInterval(t *testing.T) {
+	t.Run("via deprecated YAML", func(t *testing.T) {
+		ResetConfig(t)
+		_, err := sysconfig.New("./testdata/TestUSM-MaxHTTPMapCleanerIntervalDeprecated.yaml")
```

Comment:
FYI if you don't want to create a bunch of files, you can use `configurationFromYAML`.